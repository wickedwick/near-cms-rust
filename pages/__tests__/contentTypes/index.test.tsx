import { configure, shallow } from 'enzyme'
import ContentTypes from '../../contentTypes'
import Adapter from '@wojtekmaj/enzyme-adapter-react-17'

configure({ adapter: new Adapter() })

describe('ContentTypes', () => {
  it('shoud render', () => {
    const wrapper = shallow(<ContentTypes />)
    expect(wrapper).toMatchSnapshot()
  })
})
