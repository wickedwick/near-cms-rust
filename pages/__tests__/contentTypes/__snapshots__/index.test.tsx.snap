// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentTypes shoud render 1`] = `ShallowWrapper {}`;
